simple-avro/src/main/java/io/apicurio/registry/examples/simple/avro/SimpleAvroExample.java

```diff
@@ -61,14 +60,14 @@
  */
 public class SimpleAvroExample {
 
-    private static final String REGISTRY_URL = "http://localhost:8080/apis/registry/v2";
-    private static final String SERVERS = "localhost:9092";
+    private static final String REGISTRY_URL = "https://bu98.serviceregistry.rhcloud.com/t/88a86411-f459-4305-ba1b-d9fd81e1044f/apis/registry/v2";
+    private static final String SERVERS = "carnalca-t-c-kbn--g-ue-eacrogra.bf2.kafka.rhcloud.com:443";
```

Comment:
Probably don't want this part of the commit.

Reply:
Whoops.